commands/issue/create/issue_create.go

```diff
@@ -214,7 +214,12 @@ func createRun(opts *CreateOpts) error {
 					return err
 				}
 			}
-
+			if len(opts.Assignees) == 0 {
+				err = cmdutils.AssigneesPrompt(&opts.Assignees)
+				if err != nil {
+					return err
+				}
+			}
```

Comment:
This should be outside `if len(opts.Labels) == 0 || opts.MileStone == 0 {`

Reply:
fixed